.github/workflows/keepalive.yml

```diff
@@ -0,0 +1,27 @@
+name: Keepalive
+
+on:
+  schedule:
+    - cron: "0 0 1 * *" # Run once a month
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+
+  keepalive:
+    name: Keepalive
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: keepalive
+
+      # ensures that crons are not suspended after 60 days
+      - name: Keepalive check
+        uses: gautamkrishnar/keepalive-workflow@v1
+        with:
+          gh_token: ${{ secrets.INVENIOBOT_PAT }}
```

Comment:
I think that this variable does not exist in this repo, isn't it?
Can you point us how to create that secret? Any doc from the action's creator?

Reply:
Nothing from the action's creator, but it should be something like [this](https://michaelcurrin.github.io/code-cookbook/recipes/ci-cd/github-actions/tokens/access-token.html)

Reply:
What is technically needed to populate the access token? I.e what is left to be done for the new workflow to run?

Reply:
We'll need to generate the new personal access token (I don't have repo settings access, someone will need to do the following):
1. Go to Settings > Developer settings > Personal access tokens > classic
2. Give repo and workflow access and generate the token and copy it
3. Then go to this repo's settings > Secrets and variables > Actions
4. Add new repository secret and name it: `INVENIOBOT_PAT`

Reply:
After checking together, please move this action to v2 with `content: true`
https://github.com/gautamkrishnar/keepalive-workflow?tab=readme-ov-file#github-api-keepalive-workflow---default-for-github-actions-users